### ISSUE TITLE
508 Accessibility: Inconsistent styling leads to confusion on the purpose of a component

### ISSUE THREAD

**Issue by ghost** [asyncapi/asyncapi-react#960]
WCAG 2.0 guideline 3.2.4 Consistent Identification states "Components that have the same functionality within a set of Web pages are identified consistently".  This is Level AA.

In this block, there are multiple elements with a solid color background, but only one of them (WSS) is a button.
![image](https://github.com/asyncapi/asyncapi-react/assets/86489380/518ed145-9c3d-4f82-8173-8c20191391fc)

In this section, there are 4 components that are styled with a rectangular border.  Two are buttons and two are not.  And note these two buttons are not styled the same as the button in the example above.
![image](https://github.com/asyncapi/asyncapi-react/assets/86489380/097d445e-fde7-4c3b-b8d4-a385963ec478)

All buttons should be styled the same - preferably with the rectangular outline - and all data labels should be styled the same - preferably with the solid background.  The button styles and label styles should be different from each other.
